test/libsolidity/syntaxTests/inheritance/override/ambiguous_base_functions_overridden_in_intermediate_base.sol

```diff
@@ -0,0 +1,12 @@
+contract A {
+	function f() external virtual {}
+}
+contract B {
+	function f() external virtual {}
+}
+contract C is A, B {
+	function f() external override (A, B) {}
```

Comment:
Can you add a similar test where this one only is not implemented?

Reply:
Will add.